.gitlab/ci/bucket-upload.yml

```diff
@@ -156,7 +162,7 @@ upload-packs-to-marketplace:
         fi
 
         if [[ $GCS_MARKET_BUCKET != $GCS_PRODUCTION_BUCKET ]] && [[ -z $STORAGE_BASE_PATH ]]; then
-         STORAGE_BASE_PATH="upload-flow/builds/$CI_COMMIT_BRANCH/$CI_PIPELINE_ID/xsoar/content"
+         STORAGE_BASE_PATH="upload-flow/builds/$CI_COMMIT_BRANCH/$CI_PIPELINE_ID/content"
```

Comment:
Do we want different bucket path directories depending on whether it's `xsoar` or `mpv2`?

Reply:
it uploads to different buckets so don't think we need this indication in the path as well (duplicate information)